extensions/amp-drive-viewer/amp-drive-viewer.md

```diff
@@ -0,0 +1,67 @@
+<!---
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-drive-viewer"></a> `amp-drive-viewer`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Displays a document file supported by Google Drive.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-drive-viewer" src="https://cdn.ampproject.org/v0/amp-drive-viewer-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
+    <td>fill, fixed, fixed-height, flex-item, nodisplay, responsive</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td><a href="https://ampbyexample.com/components/amp-drive-viewer/">Annotated code example for amp-drive-viewer</a></td>
+  </tr>
+</table>
+
+[TOC]
+
+## Behavior
+
+The `amp-drive-viewer` component displays document files like Word documents,
```

Comment:
If this is specific to Drive, we should make that explicit here.

Reply:
It's a little bit confusing, since what this component does is to view anything (from Drive or not) in a Drive iframe. Maybe that's a good thing to document, since it works with everything, but for those not from Drive, puts it in a Drive iframe.